.travis.yml

```diff
@@ -117,3 +117,17 @@ jobs:
         - make edgeimage ARCH="arm64v8"
         - make edgesiteimage ARCH="arm64v8"
         - make bluetoothdevice_image
+
+  deploy:
+    provider: releases
+    api_key:
+      secure: <Enter Key Here>
```

Comment:
? something needs to be filled?

Reply:
@chendave Yes, the secure_key would allow you to publish the Debian packages on the GitHub releases. https://github.com/kubeedge/kubeedge/releases . Here this would be the key of the person having privilege to commit to the KubeEdge repo.

Reply:
If we add the key here, attackers can just copy/paste the key and make changes to the packages?